integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -396,9 +470,58 @@ public static Domain createDomainResourceWithAuxiliaryImageClusterScope(
       List<String> auxiliaryImageDomainScopeNames,
       List<String> auxiliaryImageClusterScopeNames) {
 
+    Domain domainCR =
+        createDomainResourceWithAuxiliaryImageClusterScope(domainResourceName,
+          domNamespace,
+          baseImageName,
+          adminSecretName,
+          repoSecretName,
+          encryptionSecretName,
+          replicaCount,
+          List.of(clusterName),
+          auxiliaryImagePathVolume,
+          auxiliaryImageDomainScopeNames,
+          auxiliaryImageClusterScopeNames);
+
+    return domainCR;
+  }
+
+  /**
```

Comment:
re format the javadoc with a  starting *  on each line

Reply:
changed